scaladoc/test/dotty/tools/scaladoc/site/SidebarParserTest.scala

```diff
@@ -34,6 +38,63 @@ class SidebarParserTest:
       |          - page: my-page6/my-page6/my-page6.md
       """.stripMargin
 
+  private val sidebarNoTitle =
+    """index: index.md
+      |subsection:
+      |    page: my-page1.md
+      |  - page: my-page2.md
+      |  - page: my-page3/subsection
+      |  - title: Reference
+      |    subsection:
+      |      - page: my-page3.md
+      |        hidden: true
+      |  - index: my-page4/index.md
+      |    subsection:
+      |      - page: my-page4/my-page4.md
+      |  - title: My subsection
+      |    index: my-page5/index.md
+      |    subsection:
+      |      - page: my-page5/my-page5.md
+      |  - subsection:
+      |      - page: my-page7/my-page7.md
+      |  - index: my-page6/index.md
+      |    subsection:
+      |      - index: my-page6/my-page6/index.md
+      |        subsection:
+      |          - page: my-page6/my-page6/my-page6.md
+      """.stripMargin
+
+  private val sidebarErrorNoPage =
+    """index: index.md
+      |subsection:
+      |  - title: My title
+      |  - page: my-page2.md
+      |  - page: my-page3/subsection
+      |  - title: Reference
+      |    subsection:
+      |      - page: my-page3.md
+      |        hidden: true
+      |  - index: my-page4/index.md
+      |    subsection:
+      |      - page: my-page4/my-page4.md
+      |  - title: My subsection
+      |    index: my-page5/index.md
+      |    subsection:
+      |      - page: my-page5/my-page5.md
+      |  - subsection:
+      |      - page: my-page7/my-page7.md
+      |  - index: my-page6/index.md
+      |    subsection:
+      |      - index: my-page6/my-page6/index.md
+      |        subsection:
+      |          - page: my-page6/my-page6/my-page6.md
+      """.stripMargin
+
+  private val msgNoTitle = "Error parsing YAML configuration file: 'title' is not provided."
```

Comment:
The problem is that in such a case (when a title is missing).  It is not possible to work on `r: RawInput` (,,,[],,false), even before the toSidebar function is called.

Reply:
How could we make the error message clearer? Would something like this work? “the field 'title' is missing in one of the entries of the sidebar”

Otherwise, is it possible to return more information at the place where we try to access that `title` field and then use that information when we generate the error message?

Reply:
Yes of course, so in my last commit, if a title is missing, I take the raw content and look at a `page` without a `title` before.